recipes/pynfft/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.3.2" %}
+
+package:
+  name: pynfft
+  version: {{ version }}
+
+source:
+  fn: pyNFFT-{{ version }}.tar.bz2
+  url: https://pypi.io/packages/source/p/pyNFFT/pyNFFT-{{ version }}.tar.gz
+  md5: d2457a3f43839a8acce3295c60d5e692
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - numpy >=1.6
```

Comment:
I believe there are compiled extensions in this package, right? If so, and if they depend on the numpy ABI you should change `numpy >=1.6` to `numpy x.x`. We only build `numpy >=1.9`, so the version limitation is already satisfied, but the `x.x` ensure that the right numpy, used to build, will be installed at run time by baking the numpy build string in the package.